software/control/microcontroller.py

```diff
@@ -591,6 +591,11 @@ def analog_write_onboard_DAC(self,dac,value):
         cmd[4] = value & 0xff
         self.send_command(cmd)
 
+    def set_piezo_um(self, z_piezo_um):
```

Comment:
It probably makes sense to move this here since it requires intricate knowledge of the micro dac setup.  An alternative would be to figure out how to have `AbstractStage` handle this (eg: in its Z axis)

Reply:
Yes we’ll need to think about how to deal with the different z stage options: stepper alone, stepper + piezo, direct drive stage alone (Dover Motion DOF-5 or PI V-308), piezo alone.